tools/proofers/PPage.inc

```diff
@@ -237,7 +237,7 @@ class PPage
         if ($font_size != '') {
             $font_size_string = "font-size: $font_size;";
         }
-        echo "style=\"font-family: $font_family; $font_size_string padding-left: 0.25em;\" ";
+        echo "style=\"font-family: $font_family; $font_size_string padding-left: 0.5em;\" ";
```

Comment:
I understand why `font-family` specs are here, but is there a reason that `padding-left` is here instead of `page_interfaces.less`?

Or is this one of those "Meh, it's been there for forever, it doesn't affect the styling of the themes, there's no good reason to change it, and it may very well change once the new PI is implemented, so why waste time on it?" situations?

Reply:
I think there probably isn't a good reason why it's not in `page_interfaces.less`. Specifying it for  `id text_data` could be better then it would work for the quiz also. I'd have to check what else text_data is used for so I was just being lazy.

Reply:
grepping `id='text_data'` revealed another textarea in `tools/project_manager/handle_bad_page.php` so it would perhaps be good to put the padding in page_interfaces and remove the padding from PPage.inc. and quiz/generic/proof.php.
(Note to self: I induced some other errors while running fix bad page so should look at them later)